docs/_Docs_ReputationMining.md

```diff
@@ -97,16 +97,21 @@ The reputation mining client will answer queries for reputation scores locally o
 http://127.0.0.1:3000/{reputationState}/{colonyAddress}/{skillId}/{userAddress}
 ```
 
-The oracle should be able to provide responses to any valid reputation score in all historical states, as well as the current state.
+An instance of the oracle is available for reputation queries against `goerli` or `mainnet`:
+```
+https://colony.io/reputation/{goerli|mainnet}/{reputationState}/{colonyAddress}/{skillId}/{userAddress}
```

Comment:
This works but maybe we can use `{network}` instead of `{goerli|mainnet}` to make this a valid code snippet. `goerli` and `mainnet` are clearly defined as the supported networks above.